djangogirls/settings.py

```diff
@@ -47,8 +47,8 @@
     'django_date_extensions',
     'storages',
     'markdown_deux',
-
     'django_countries',
+    'crispy_forms',
```

Comment:
adding in django-crispy-forms (although the code's only using the very basic form of it at the moment)